cookiecutter/repository.py

```diff
@@ -41,6 +41,20 @@ def expand_abbreviations(template, abbreviations):
     return template
 
 
+def valid_repository(repo_directory):
```

Comment:
While this works great for checking if the `cookiecutter.json` file exists, it doesn't confirm that the `cookiecutter.json` module is in fact valid JSON. Indeed, later on all we do is `json.load`, providing a non-descriptive error message. Perhaps this function could address that as well?